paper/paper.md

```diff
@@ -33,13 +33,14 @@ Ipyannotator is a library developed to provide a solution to this problem. Ipyan
 
 Jupyter is one of the most popular tools for data science [@wang2019how] and is currently used by more than 7850000 public repositories on GitHub [@parente2022nbestimate]. Ipyannotator is a tool developed to be used on Jupyter Notebook, allowing researchers and developers to integrate the library into their ML projects quickly and easily. The solution, however, is not limited to research and developement teams. Since the Ipyannotator runs also on a web server, it can by used for annotation purposes by any user.
 
-[@lahtinen2021brima] developed an annotator as a browser plugin, [@dutta2019via] built an annotator as a program that runs on the browser, and [@bernal2019gtcreator] developed a desktop application for domain-specific medical image annotation. The previous annotation tools are restricted to fixed data types and a specific type of annotation. Since Ipyannotator runs on top of Jupyter notebooks and due to the dynamic nature of the Python language, extra features and customization can be easily developed by the data science team, creating an "infinitely hackable annotation framework".
+[@lahtinen2021brima] developed an annotator as a browser plugin, [@dutta2019via] built an annotator as a program that runs on the browser, and [@bernal2019gtcreator] developed a desktop application for domain-specific medical image annotation. The previous annotation tools are restricted to fixed data types and a specific type of annotation. Ipyannotator was designed to be executed in Jupyter notebook and allow users to change its iterations on runtime, allowing extensions and customization, creating an "infinitely hackable annotation framework" [^1].
 
-# Infinitely hackable annotation framework
+[^1]: "infinitely hackable" is one of the key design principles used. Even through using very thin python wrappers e.g. [@ipycanvas2022] for the JavaScript Web Canvas API, ipyannotator still has to obey the finite limitations of the python language.
 
+# Infinitely hackable annotation framework
```

Comment:
```suggestion
# Infinitely hackable[^1] annotation framework
```

@itepifanio Does this render properly? Should we add the footnote here as well?

Reply:
> Does this render properly?

@ibayer it doesn't render very well. For some reason, it creates a second footnote:

![Screenshot from 2022-07-15 18-26-58](https://user-images.githubusercontent.com/6730205/179313159-077e7f82-51d2-420c-86b3-9cded7c03491.png)

> Should we add the footnote here as well?

Since the same phrase it's mentioned before with a footnote we may not need it.

Reply:
Thanks for checking, let's not add it then.